protocol/common/endpoints.go

```diff
@@ -20,7 +20,9 @@ const (
 	URL_QUERY_PARAMETERS_SEPARATOR_OTHER_PARAMETERS = "&"
 	IP_FORWARDING_HEADER_NAME                       = "X-Forwarded-For"
 	PROVIDER_ADDRESS_HEADER_NAME                    = "Lava-Provider-Address"
-	BLOCK_PROVIDERS_ADDRESSES_HEADER_NAME           = "Lava-Providers-Block"
+	// these headers need to be lowercase
+	BLOCK_PROVIDERS_ADDRESSES_HEADER_NAME = "lava-providers-block"
```

Comment:
PROVIDER_ADDRESS_HEADER_NAME is capitalized and is not, should it be consistant?